src/main/resources/data/computercraft/lua/rom/programs/turtle/craft.lua

```diff
@@ -11,9 +11,9 @@ end
 local tArgs = { ... }
 local nLimit = tonumber(tArgs[1])
 
-if not nLimit and tArgs[1] then
+if not nLimit and tArgs[1] ~= "all" then
     local programName = arg[0] or fs.getName(shell.getRunningProgram())
-    print("Usage: " .. programName .. " [number]")
+    print("Usage: " .. programName .. " [all/number]")
```

Comment:
As the amount is required, it should be `<number>` now - I think `all|<number>` makes the most sense, or have two separate usage lines.

Reply:
Alright, it's now using `all|<number>`